content_scripts/normal.js

```diff
@@ -510,8 +510,11 @@ function createNormal() {
                 elm.smoothScrollBy(x, y, d);
             } else {
                 document.dispatchEvent(new CustomEvent('surfingkeys:scrollStarted'));
-                elm.scrollLeft = elm.scrollLeft + x;
-                elm.scrollTop = elm.scrollTop + y;
+                elm.scrollBy({
+                    'behavior': 'instant',
```

Comment:
There is no value `instant` for `behavior`, according to https://developer.mozilla.org/en-US/docs/Web/API/ScrollToOptions/behavior

Have you tested this change with both Chrome and Firefox?

Reply:
@brookhong 
This option works both on the latest chrome and Firefox. On Firefox, instant truly is the same as auto, but on chrome, auto does not prevent smooth scrolling on the above site (and only instant does that). 

There is either a bug in chrome (as instant does not mirror the behavior of auto) or in the spec. Since both browers reject incorrect values with errors, and because auto is non-specific about behavior, I think the spec is incorrect here.

I'm not aware of another way to force non-smooth scrolling through js that works on the given site.

FWIW, Vimium seems to use this option as well: https://github.com/philc/vimium/blob/8c9404f84fd4310b89818e132c0ef0822cbcd059/content_scripts/scroller.coffee#L60

See https://github.com/w3c/csswg-drafts/issues/3497 and https://github.com/w3c/csswg-drafts/issues/2146

Reply:
Great, thanks for the information.